README.md

```diff
@@ -1127,6 +1127,53 @@ flags to Lua.
 * `"TX"`: n/a.
 * `"REDIRECT"`: n/a.
 
+### xtable
+
+The `xtable` library provides support for developing netfilter xtable extensions.
+
+#### `xtable.match(match_opts)`
+
+_xtable.match()_ returns a new `xtable` object for matching packets.
+This function receives the following arguments:
+* `match_opts`: a table containing the following fields:
+  * `name`: string representing the xtable extension name.
+  * `revision`: integer representing the xtable extension revision.
+  * `family`: address family, one of [xtable.family](https://github.com/luainkernel/lunatik#xtablefamily).
+  * `proto`: protocol number.
+  * `match` : function to be called for matching packets. It receives the following arguments:
```

Comment:
If we are going to name this field `match`, we could have a single function for adding hooks (e.g., `xtable.new()`) and take the decision based on such field.. otherwise, I would change this field name (e.g., `callback`)